lib/rubocop/cop/rspec/rails/http_status.rb

```diff
@@ -1,213 +1,60 @@
 # frozen_string_literal: true
 
-require 'rack/utils'
```

Comment:
We can now remove rack-utils from the gemspec?

Reply:
Yes! Thank you. I remove it.